incubator-tools/pii_synthetic_redaction_tool/pii_synthetic_redaction_tool.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<p>Can this notebook be combined with the previous Pii redaction tool and just have an extra function for Synthetic data?</p><p> </p><p>Looks like a lot is the same between them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done